docs/introduction/installation.md

```diff
@@ -187,11 +187,49 @@ After you've installed the Community Apps Plugin, you can install Homarr from th
 
 After you've installed Homarr, you can find it under the tab "Docker".
 
+The official support thread is located [here](https://forums.unraid.net/topic/123478-support-smartphonelover-homarr/), but we prefer to communicate over GitHub and Discord instead.
+
 ---
 
-## Building from source
+## Installation on Synology
+The installation process is also quite easy and fast on Synology devices.
+Since this method is used less often, we recommend you the thiry party guide of mariushosting.
+
+https://mariushosting.com/how-to-install-homarr-on-your-synology-nas/
+
+:::note
+
+This is a third party guide - we do not garantuee if it's up to date nor offer support. Use at your own risk.
+
+:::
```

Comment:
```suggestion
The installation process is quite easy and fast on Synology devices.
Since this method is used less often, we recommend you to follow this guide written by mariushosting.

https://mariushosting.com/how-to-install-homarr-on-your-synology-nas/

:::note

This is a third party guide! It might not be up-to-date, nor do we offer official support.

:::
```